app/templates/snips/studentHistoryModal.html

```diff
@@ -213,6 +213,7 @@
       <div class = "col-sm-{{12/buttonState.num_buttons}}">
 
       <a href="#"><button class="btn btn-primary" id="emailLabor">Email Labor</button></a>
+      <button type="button" class="btn btn-primary" data-dismiss="modal" onclick="submitToBanner({{statusForm.laborStatusFormID}})">Resubmit Form</button>     
```

Comment:
Do you see how all of the other buttons use the buttonState object to determine if the button should display? This is necessary because it doesn't make sense to have a Resubmit button on every form, and certainly not for every user. Students should not be able to submit their forms to banner, for example.

Reply:
Has not been resolved yet